models/yang/sonic/common/sonic-common.yang

```diff
@@ -0,0 +1,41 @@
+
+module sonic-common {
+	namespace "http://github.com/Azure/sonic-common";
+	prefix scommon;
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC VLAN";
+
+	revision 2019-05-15 {
+		description
+			"Initial revision.";
+	}
+
+	typedef tagging_mode {
+		type enumeration {
+			enum untagged;
+			enum tagged;
+			enum priority_tagged;
+		}
+	}
+
+	typedef admin-status {
+		type enumeration {
+			enum up;
+			enum down;
+		}
+	}
+
+
+	container operation {
```

Comment:
I can add description. This schema is used internally by CVL and REST API will not be generated for this. We can make the type as ENUM instead of string.

I thought about read-only again. But since its value will be changing for each operation. it is not appropriate to make it as read-only. Moreover, this is used only for validation and its valued does not go to Config DB.

Reply:
Resolved. Modified as enum.